docs/moments.rst

```diff
@@ -62,6 +62,28 @@ plotting packages such as APLpy::
     >>> f.show_colorscale()  # doctest: +SKIP
     >>> f.save('moment_0.png')  # doctest: +SKIP
 
+The equation for the N-th moment, where N is usually an integer between -1 and some arbitrary
+positive integer say 11, is as follows:
+
+.. math:: M_N = \frac{\int I (l - M_l)^N dl}{M_0}
+
+Descriptions for the twelve most common moments used are as follows:
+
+moments=-1 - the mean value of the spectrum  
```

Comment:
This is incorrect.  `moment=-1` is not defined and is not supported by spectral-cube.